### DISCUSSION TITLE
Memories and modules

### DISCUSSION THREAD

**Discussion by clustercd** [KoljaB/LocalAIVoiceChat#11] [Q&A]
Hi, I tried and reached to ran the excellent Linguflex2, but unfortunately it’s too complex and big for me as there’s many options I don’t really need. The question here it’s how and what to select in linguflex 2 to be able to add the module management only in localAivoice? Also I still have a bit latency ( small break each 3-4 seconds) even with cuda. May I asking you if you have some tips to set up to improve the fluency?
Thx !

**Comment by KoljaB** [comment]
You can disable modules in Linguflex by going into the modules directory and add a underscore in front of the module name. So if you want to disable the "mail" module you rename the directory to "_mail" and linguflex will ignore it. So you could disable all modules except for "brain", "listen" and "speech" (and maybe "mimic") and have a very slim linguflex which does not need to configure much stuff.

If you still have breaks in speech generation I assume you are using the Coquiengine, which needs a fast GPU for realtime tts. It is very possible that your GPU is just not fast enough to do this, I would just choose another speech provider then, not coqui. Nothing I can do here.

Also I probably won't do changes to LocalAIVoiceChat. This was just a super slim demo showcase, most of my work goes into linguflex being my main project.

**Reply by KoljaB** [reply]
I will implement a "full sentences" flag into Linguflex soon which allows to synthesize a full speech sentence before playing it out, so on slower systems you at least don't have breaks midsentence. But still you will have breaks after each sentence.
